docs/docs/build-caching.md

```diff
@@ -62,15 +62,15 @@ exports.onPostBuild = async function({ cache, store, graphql }, { query }) {
+- Si `gatsby-config.js` cambia, por ejemplo, un nuevo complemento es agregado o modificado
+- Si `gatsby-node.js` cambia, por ejemplo, si invocas una API Node nueva, o cambia una llamada a `createPage`
 
-## Conclusion
+## Conclusión
 
-With the cache API you're able to persist data between builds, which is really helpful while developing a site with Gatsby (as you re-run `gatsby develop` really often). Performance-heavy operations (like image transformations) or downloading data can slow down the bootstrap of Gatsby significantly and adding this optimization to your plugin can be a huge improvement to your end users. You can also have a look at the following examples who implemented the cache API: [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-contentful/src/download-contentful-assets.js), [gatsby-source-shopify](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-shopify/src/nodes.js#L23-L54), [gatsby-source-wordpress](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-wordpress/src/normalize.js#L471-L537), [gatsby-transformer-remark](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-transformer-remark/src/extend-node-type.js), [gatsby-source-tmdb](https://github.com/LekoArts/gatsby-source-tmdb/blob/e12c19af5e7053bfb7737e072db9e24acfa77f49/src/add-local-image.js).
+Con la API caché, vas a ser capaz de persistir datos e información entre diferentes compilaciones, lo cual es bastante útil cuando desarrollas un sitio con Gatsby (ya que corres `gatsby develop` muy frecuentemente). Las operaciones que requieren de gran rendimiento (como por ejemplo, transformaciones de imágenes) o la descarga frecuente de datos, pueden presentar una ralentización significativa en el desempeño de Gatsby, por lo cual agregar esta optimización de caché puede resultar en una mejora notable para los usuarios de tu sitio. También puedes echar un vistazo a los siguientes ejemplos que utilizan la API caché: [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-contentful/src/download-contentful-assets.js), [gatsby-source-shopify](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-shopify/src/nodes.js#L23-L54), [gatsby-source-wordpress](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-source-wordpress/src/normalize.js#L471-L537), [gatsby-transformer-remark](https://github.com/gatsbyjs/gatsby/blob/7f5b262d7b5323f1a387b8b7278d9a81ee227258/packages/gatsby-transformer-remark/src/extend-node-type.js), [gatsby-source-tmdb](https://github.com/LekoArts/gatsby-source-tmdb/blob/e12c19af5e7053bfb7737e072db9e24acfa77f49/src/add-local-image.js).
```

Comment:
(ya que corres `gatsby develop` muy frecuentemente) => (ya que ejecutas `gatsby develop` frecuentemente)

Reply:
Thanks for the suggestions @nucliweb !

I think you're right about plugins, but since we don't have that word in the glossary of terms would be good to add it before using it. But yeah, I'd prefer to use `plugins` as it's a more common word and understandable for programmers.

Regarding the use of `correr`: I think it's common use among Spanish programmers as it's less formal than `ejecutar`. Since it says `run` instead of `execute`, I opted for `correr`. I'm open to change it though!

Reply:
What do you think about `correr` vs `ejecutar` @horacioh and @Darking360 ?

And in reference to the plugin, we'll add `plugin` to the glossary, because is used a lot in the documentation and we have other docs translated with `plugin`

Reply:
Hello, and for me is "ejecutar" since "correr" sounds like anglicism 😅

Reply:
OK, sounds good, I'll change it 👍 .

I think it's a common term among Spanish developers though 😅 :

![image](https://user-images.githubusercontent.com/5914822/66704048-2397ca80-ecef-11e9-9124-76e6e017dd7d.png)
![image](https://user-images.githubusercontent.com/5914822/66704049-2d213280-ecef-11e9-9ce3-e19532a62144.png)

Reply:
OK, sounds good, I'll change it 👍